packages/graphql/tests/e2e/subscriptions/create-relationship.e2e.test.ts

```diff
@@ -333,7 +335,7 @@ subscription SubscriptionPerson {
             })
             .expect(200);
 
-        await delay(2);
+        await delay(4);
```

Comment:
Not ideal, but maybe add a comment mentioning that we need a delay to ensure the non-existance of events here?